src/domains/ecommerce-common/02_security.tf

```diff
@@ -539,6 +539,18 @@ resource "azurerm_key_vault_secret" "npg_mybank_psp_keys" {
   value        = "<TO UPDATE MANUALLY ON PORTAL>"
   key_vault_id = module.key_vault.id
 
+  lifecycle {
+    ignore_changes = [
+      value,
+    ]
+  }
+}
+
+resource "azurerm_key_vault_secret" "wallet_storage_connection_string" {
+  name         = "wallet-storage-connection-string"
+  value        = "<TO UPDATE MANUALLY ON PORTAL>"
```

Comment:
you could use `primary_connection_string` of [module output](https://github.com/pagopa/terraform-azurerm-v3/blob/97e011d8a6bebee22aaad8e02a40700fd7fca661/storage_account/outputs.tf#L9C9-L9C34) .

Reply:
The wallet storage is created on wallet domain side, so the storage connection string secret in `wallet-common` now reference the output of `wallet_storage` module. The same connection string is needed on eCommerce domain side, which cannot reference directly the storage module (unlike to reference it through data or something else), we can consider the connection string as "external secret" from eCommerce domain? (something like an api key) or create a reference via "data"?